CONTRIBUTING.md

```diff
@@ -4,130 +4,139 @@ Welcome to the Hacktoberfest repository organized by the Microsoft Learn Student
 
 ## How to Install Dependencies and Work on the Project Locally
 
-From your terminal:
+1. **Clone the Repository:**
 
-Clone your forked repository and name it `AI-Hacktober-MLSA`
+   From your terminal, clone your forked repository and name it `AI-Hacktober-MLSA`.
 
-```
-# Replace {user_name} with your GitHub username
-git clone --depth 1 https://github.com/{user_name}/AI-Hacktober-MLSA.git
-```
-> This makes sure you clone only the most recent commit and all files to avoid cloning every commit history as image files have been resized and hence will only take space on your local machine for what you don't need.
+   ```bash
+   # Replace {user_name} with your GitHub username
+   git clone --depth 1 https://github.com/{user_name}/AI-Hacktober-MLSA.git
+   ```
 
-Create a virtual environment with the name (`AI-Hacktober-MLSA`)
+   > This ensures you clone only the most recent commit and all files to avoid unnecessary commit history.
```

Comment:
```suggestion
   > This ensures you clone only the most recent commit and all files to avoid unnecessary commit history (image files that have been resized).
```